### PR TITLE
React/week2

### PR DESCRIPTION
**Pull request by mahakarzoun**

### CODE REVIEW

**Review by nuhana — COMMENTED** [review]
nice work 👍

File: react/week2/my-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import './App.css';
+import Main from './containers/Main';
+
+function App() {
+  return (
+    <div className="App">
+      <Main />
```
Comment (nuhana) [inline react/week2/my-app/src/App.js:10]: can you choose more spesific component name

File: react/week2/my-app/src/database/Tasks.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const tasks = [
```
Comment (nuhana) [inline react/week2/my-app/src/database/Tasks.js]: you can do it with json file

File: react/week2/my-app/src/containers/Main.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            updateTasks(newTasks)
+            Input.current.value = "";
+        }
+    }
+
+
+    return (
+        <div>
+            <div className="todo">
+                <button onClick={addTask.bind(this)}> add new task </button>
```
Comment (nuhana) [inline react/week2/my-app/src/containers/Main.js]: Did you used 'this' in your code?

**Review by nuhana — CHANGES_REQUESTED** [review]

File: react/week2/my-app/src/containers/Main.js (hunk 3 above)
Comment (mahakarzoun) [reply react/week2/my-app/src/containers/Main.js]: yes. do you have any other opinion about that?

Comment (nuhana) [reply react/week2/my-app/src/containers/Main.js]: you didn't use it why are you binding ? just remove it

**Pull request merged**